runtime/compiler/compile/J9Compilation.cpp

```diff
@@ -1144,9 +1148,36 @@ J9::Compilation::addAsMonitorAuto(TR::SymbolReference* symRef, bool dontAddIfDLT
    }
 
 TR_OpaqueClassBlock *
-J9::Compilation::getClassClassPointer()
+J9::Compilation::getClassClassPointer(bool isVettedForAOT)
    {
-   return _ObjectClassPointer ? self()->fe()->getClassClassPointer(_ObjectClassPointer) : 0;
+   if (!isVettedForAOT)
+      return _ObjectClassPointer ? self()->fe()->getClassClassPointer(_ObjectClassPointer) : 0;
+
+   if (_aotClassClassPointerInitialized)
+      return _aotClassClassPointer;
```

Comment:
This might be nit-picky, but do we need `_aotClassClassPointerInitialized`? Isn't it sufficient to know that if `_aotClassClassPointer` is NULL it hasn't been initialized?

Reply:
It might be unavailable even though we set `isVettedForAOT=true`